src/Umbraco.Web.UI.Client/src/views/propertyeditors/grid/editors/embed.html

```diff
@@ -1,10 +1,10 @@
 <div ng-controller="Umbraco.PropertyEditors.Grid.EmbedController">
 
-	<div class="umb-editor-placeholder" ng-click="setEmbed()" ng-if="trustedValue === null">
+	<div class="umb-editor-placeholder" ng-click="setEmbed()" ng-if="hasEmbed() === false">
 		<i class="icon icon-movie-alt"></i>
 		<div class="help-text"><localize key="grid_clickToEmbed">Click to embed</localize></div>
 	</div>
 
-	<div ng-if="control.value" ng-bind-html="trustedValue"></div>
+	<div ng-if="control.value" ng-bind-html="getEmbed()"></div>
```

Comment:
binding to a method means that it's going to execute this method on every digest, i think there was a reason this was previously bound to a property, binding to a method is expensive

Reply:
I know, but the request goes on making it work in Split-view. I will change to a $watch method instead.